server/src/main/java/org/elasticsearch/rest/action/cat/RestHealthAction.java

```diff
@@ -70,6 +70,8 @@ protected Table getTableWithHeader(final RestRequest request) {
         t.addCell("pri", "alias:p,shards.primary,shardsPrimary;text-align:right;desc:number of primary shards");
         t.addCell("relo", "alias:r,shards.relocating,shardsRelocating;text-align:right;desc:number of relocating nodes");
         t.addCell("init", "alias:i,shards.initializing,shardsInitializing;text-align:right;desc:number of initializing nodes");
+        t.addCell("unassign.primary",
+                  "alias:up,shards.unassigned.primary,shardsUnassignedPrimary;text-align:right;desc:number of unassigned primary shards");
```

Comment:
I don't think this aligns with our Checkstyle configuration. I don't know if you already got the `Adapter for Eclipse Code Formatter` plugin working already, but in the meanwhile, you can run `gradle spotlessApply` to format all code in compliance with our Checkstyle configuration.

Reply:
Have got this all setup now and thanks for the tip on spotlessApply!